pkg/cmd/helmfile/move/move.go

```diff
@@ -311,3 +315,21 @@ func (o *Options) moveFilesToClusterOrNamespacesFolder(dir, ns, releaseName, cha
 	}
 	return nil
 }
+
+// BuildOutputDirPath Builds output directory path. Returns: directory path where file should be placed
+func (o *Options) BuildOutputDirPath(node *yaml.RNode, path, ns, pathName string) string {
+	return filepath.Join(o.NamespacesDir, GetTargetNamespace(node, path, ns), pathName)
+}
+
+func GetTargetNamespace(node *yaml.RNode, path, ns string) string {
```

Comment:
Will this be used outside of jx-gitops? If not we dont need to export these, unless u have it this way to add tests for these functions ...

Reply:
Same here as https://github.com/jenkins-x-plugins/jx-gitops/pull/835#discussion_r826643000

Reply:
I don't think it would be used outside. Fixed.